packages/wrangler/src/dev.tsx

```diff
@@ -228,6 +232,11 @@ function useLocalWorker(props: {
       }
 
       await waitForPortToBeAvailable(port, { retryPeriod: 200, timeout: 2000 });
+      if (props.public) {
+        throw new Error(
```

Comment:
index.tsx has a `NotImplementedError` which we could use here. But you'd probably need to move it into a shared file so we don't get a cyclic dependency.

Reply:
I'll get it when I come back to write tests for `dev`.